server/apiCalls/getFilteredData.js

```diff
@@ -48,8 +48,6 @@ function _parseCourseDefaultInformation(courseDetails, language) {
     course_recruitment_text: parseOrSetEmpty(course.recruitmentText, language, true),
     course_required_equipment: parseOrSetEmpty(course.requiredEquipment, language),
     course_suggested_addon_studies: parseOrSetEmpty(course.addOn, language),
-    course_supplemental_information_url: parseOrSetEmpty(course.supplementaryInfoUrl, language),
```

Comment:
You need to remove the usage of `course_supplemental_information_url` and `course_supplemental_information_url_text` in `CourseSectionList.jsx` as well.